openshift/e2e-tests-openshift.sh

```diff
@@ -183,24 +184,31 @@ EOF
 function install_knative(){
   header "Installing Knative"
 
-  # Create knative-serving namespace, needed for imagestreams
-  oc create namespace $SERVING_NAMESPACE
+  echo ">> Patching Knative Serving CatalogSource to reference CI produced images"
+  CURRENT_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+  RELEASE_YAML="https://raw.githubusercontent.com/openshift/knative-serving/${CURRENT_GIT_BRANCH}/openshift/release/knative-serving-ci.yaml"
+  sed "s|--filename=.*|--filename=${RELEASE_YAML}|"  openshift/olm/knative-serving.catalogsource.yaml > knative-serving.catalogsource-ci.yaml
```

Comment:
Did you consider building it from the operator itself? `./hack/release.sh`

Reply:
We did, this is what our call with @zroubalik a while ago was about if you remember. We need the file to be ready at some URL.

The alternative is to generate it on the fly, stuff it into a configMap, mount the configMap to the operator to magically make the file appear. That can work in principle, we agreed on going the route with the checked in file for now though.

Reply:
Gotcha. I remember.